forge/ee/emailTemplates/LicenseGrace.js

```diff
@@ -0,0 +1,40 @@
+// Inserts:
+// - user.name
+// - forgeURL
+// - days
+
+module.exports = {
+    subject: 'Your FlowForge license has entered the grace period',
```

Comment:
If the decision via #1744 is to let things keep running after license expiry, then the grace period does not serve a lot of purpose - as there will be no significant change in behaviour.

We should send `LicenseExpired` - no need for a separate email talking about grace periods. The grace period is an internal concept we may choose to make use of. For the user, the fact is their license has expired.

Reply:
resolved in [7f3c7cc](https://github.com/flowforge/flowforge/pull/1757/commits/7f3c7cc6dd8691259b2b0758d050a5b6abd140ff)